### ISSUE TITLE
helmfile deps is failing with Chart.lock: no such file or directory with helm 3.0.2

### ISSUE THREAD

**Issue by Nitive** [roboll/helmfile#1052]
`helmfile deps` is failing with `Chart.lock: no such file or directory` with helm 3.0.2.
It happens because helmfile tries to access Chart.lock which is not there in 3.0.2.
The behavior was changes in https://github.com/helm/helm/pull/7009 and now charts without `apiVersion: v2` are treated  as helm 2 carts and generate requirements.lock instead

###  Error:
`in ./helmfile.yaml: unable to update deps: unable to resolve 1 deps: open /var/folders/k_/361cq75j2sgf6dxm6625422m0000gn/T/79305915    5/Chart.lock: no such file or directory`

### Steps to reproduce:

[Repo with reproducible example](https://github.com/Nitive/helmfile-deps-helm-301-bug)

1. Clone repo with example: `git clone git@github.com:Nitive/helmfile-deps-helm-301-bug.git`
2. Install helm 3.0.1 and helm 3.0.2 with `./download-helm.sh linux-amd64` or `./download-helm.sh darwin-amd64`
3. Run `helmfile deps` with helm 3.0.1 — success

```
$ ./test-helm301.sh

Adding repo stable https://kubernetes-charts.storage.googleapis.com
"stable" has been added to your repositories

Updating repo
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "csssr" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈ Happy Helming!⎈

Updating dependency /var/folders/k_/361cq75j2sgf6dxm6625422m0000gn/T/041766349
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "csssr" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading docker-registry from repo https://kubernetes-charts.storage.googleapis.com
Deleting outdated charts
```

4. Run `helmfile deps` with helm 3.0.2 — error
```
$ ./test-helm302.sh

Adding repo stable https://kubernetes-charts.storage.googleapis.com
"stable" has been added to your repositories

Updating repo
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "csssr" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈ Happy Helming!⎈

Updating dependency /var/folders/k_/361cq75j2sgf6dxm6625422m0000gn/T/793059155
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "csssr" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading docker-registry from repo https://kubernetes-charts.storage.googleapis.com
Deleting outdated charts

in ./helmfile.yaml: unable to update deps: unable to resolve 1 deps: open /var/folders/k_/361cq75j2sgf6dxm6625422m0000gn/T/793059155/Chart.lock: no such file or directory
```

**Comment by mumoshu** [comment]
@Nitive Thanks a lot for the detailed report!

This should be fixed on the helmfile side.

**Comment by mumoshu** [comment]
@Nitive I've fixed this in #1056 which is included since v0.98.1.

The example repository did help me a lot for reproducing the problem and testing the fix.

Thanks again for your support!

**Comment by Nitive** [comment]
Thanks for a quick fix!
